authres_test.go

```diff
@@ -26,7 +26,20 @@ func TestParseAuthenticationResults(t *testing.T) {
 					{
 						Method:     "spf",
 						Result:     "pass",
-						Properties: []string{"smtp+++mailfrom+++example.net"},
+						Properties: []Property{Property{"smtp", "mailfrom", "example.net"}},
```

Comment:
Not really an issue, but there is no need to duplicate `Property` here. Go's compiler is smart enough to guess the type of a slice's item.